packages/edit-post/src/components/options-modal/index.js

```diff
@@ -55,6 +56,11 @@ export function OptionsModal( { isModalActive, isViewable, closeModal } ) {
 					) }
 				/>
 			</Section>
+			<Section title={ __( 'Keyboard options' ) }>
+				<EnableKeepCaretInBlock
```

Comment:
Can we use `EnableFeature` instead to enaable persisting the setting in local storage and update the EditorProvider to pass down that as a setting to `block-editor`  instead? (similar to `mostUsedBlocks` for instance)